lens-macro/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "lens-macro"
```

Comment:
We might consider calling this `lens-derive`, which is a common naming convention in other crates, but this does contain a non-derive macro as well so this seems okay too.

Reply:
I went with druid-derive-lens to match the data one